app/schema/mutations/application/withdrawApplication.ts

```diff
@@ -0,0 +1,24 @@
+
+import { graphql } from 'react-relay';
+import type { withdrawApplicationMutation } from '__generated__/withdrawApplicationMutation.graphql';
+import useMutationWithErrorMessage from '../useMutationWithErrorMessage';
+
+const mutation = graphql`
+  mutation withdrawApplicationMutation($input: WithdrawApplicationInput!) {
+    withdrawApplication(input: $input) {
+      application {
+        updatedAt
+        ccbcNumber
+        intakeId
```

Comment:
you'll want to fetch the status here. Ideally you'd have a fragment here, so that the component using the mutation gets all the data it needs

Reply:
I've added in status, and you mean to create a fragment for the dashboard and then reuse that fragment here right?

Reply:
Correct, that would ensure that the mutation return and the fragment that is displayed when triggering the mutation are always in sync